plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_ignored_rules: 'D2, I2, T4B'
+});
+```
+
+### `a11ychecker_issue_url_callback`
+
+The `a11ychecker_issue_url_callback` option is used to change the target URL for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) in the Accessibility Checker dialog. By default, the "more info" links will point to `{{site.url}}/docs/plugins/a11ychecker/#<ruleId>`, such as `{{site.url}}/docs/plugins/a11ychecker/#D1`. This option can be used to set the target URL to a page or pages outside {{site.url}}/docs/.
+
+**Type:** `Function`
+
+**Default Value:** `undefined`
+
+#### Example: Using `a11ychecker_issue_url_callback`
+
+This example shows how to change the link for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) on the Accessibility Checker dialog to point to anchors at `www.example.com/tinymce/a11ychecker/more_info`.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_issue_url_callback: function (ruleId) {
+    return `https://www.example.com/tinymce/a11ychecker/more_info#${ruleId}`
+  };
 });
 ```
 
 ### `a11ychecker_level`
 
 This configuration option sets the [WCAG level](https://www.w3.org/TR/WCAG20/#conformance) to use when checking issues.
 
-For example, the "Text must have a contrast ratio of at least ..." rule when using **AA** will trigger when the contrast ratio is less than 4.5:1, however when using **AAA** the rule will trigger when the ratio is less than 7.0:1.
+For example, the "Text must have a contrast ratio of at least ..." rule:
+
+* When using **AA**, Accessibility Checker will check that the contrast ratio is not less than 4.5:1.
+* When using **AAA**, Accessibility Checker will check that the contrast ratio is not less than 7.0:1.
 
 **Type:** `String`
 
 **Default value:** `aa`
 
 **Possible Values:** `a`, `aa`, `aaa`
 
-#### Example: a11ychecker_level
+#### Example: Using `a11ychecker_level`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_level: 'aaa'
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_level: 'aaa'
 });
 ```
 
 {% include misc/plugin-toolbar-button-id-boilerplate.md %}
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
-## API
+## APIs
 
-Accessibility Checker exposes couple of methods that can be called directly.
+The {{pluginname}} plugin provides the following APIs.
 
 ### `toggleaudit()`
 
-Triggers accessibility dialog with the results of the audit and options to correct the problems, if any.
+Opens the accessibility checker dialog with the results of the audit and options to correct the problems, if any.
```

Comment:
Do we care to mention that if the dialog is open, calling this closes it?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1699/commits/57201f8a1b8a53d3c0861a819b68e45f66e68798